src/details/ArborX_MinimumSpanningTree.hpp

```diff
@@ -13,7 +13,8 @@
 #define ARBORX_MINIMUM_SPANNING_TREE_HPP
 
 #include <ArborX_AccessTraits.hpp>
-#include <ArborX_DetailsKokkosExtScopedProfileRegion.hpp>
+#include <ArborX_DetailsKokkosExtArithmeticTraits.hpp>
+#include <ArborX_DetailsKokkosExtMinMaxOperations.hpp>
```

Comment:
What is the deal with these?  Oversight?

Reply:
Yes, probably happened during a rebase.